### PR TITLE
Fix event list styling issues

### PR DESCRIPTION
**Pull request by maurizi**

### CODE REVIEW

File: src/nyc_trees/js/src/event_list_page.js [hunk 1]
```diff
@@ -64,12 +66,15 @@ $(dom.mapToggle).one('shown.bs.tab', function() {
...
-        return (css.match (/(^|\s)tab-status-\S+/g) || []).join(' ');
-    });
-    $('.page-map').addClass('tab-status-'+selectedTab);
+    var selectedTab = $(e.currentTarget).attr('href');
+    $(dom.mapContainer).removeClass('tab-status-list tab-status-map-tab');
+
+    if (selectedTab === '#map-tab') {
+        $(dom.mapContainer).addClass('tab-status-map-tab');
+    } else if (selectedTab === '#list') {
+        $('.page-map').addClass('tab-status-list');
```
Comment (kdeloach) [inline src/nyc_trees/js/src/event_list_page.js]: Consider using `dom.mapContainer` instead of `.page-map`.

Comment (maurizi) [inline src/nyc_trees/js/src/event_list_page.js]: Oops, good catch!

**Comment by kdeloach** [comment]
+1

**Comment by steventlamb** [comment]
+1, tested, works

**Comment by steventlamb** [comment]
Scratch that, does not work, but is not necessarily the result of this pull. I'm looking into this, and will update as I go.

**Comment by steventlamb** [comment]
Opened a PR on @maurizi's branch that should make this work.

**Pull request merged**